src/test/java/de/sakpaas/backend/service/SearchServiceTest.java

```diff
@@ -0,0 +1,141 @@
+package de.sakpaas.backend.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import de.sakpaas.backend.HappyHamsterTest;
+import de.sakpaas.backend.model.CoordinateDetails;
+import de.sakpaas.backend.model.Location;
+import de.sakpaas.backend.model.LocationDetails;
+import de.sakpaas.backend.model.SearchRequest;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class SearchServiceTest extends HappyHamsterTest {
+
+  @MockBean
+  private LocationService locationService;
+
+  @Autowired
+  private SearchService searchService;
+
+  @Test
+  public void testFilterUnwantedLocationsByName() {
+    final String[] knownBrands = {"1"};
```

Comment:
Um den Test aussagekräftiger zu machen könnte man eine Brand "LIDL" hinzufügen und nach "LiDl" suchen.

Reply:
Findest du? Reicht doch auch so, oder?

Reply:
Sonst wäre ja das mit der Groß- und Kleinschreibung aufgefallen, oder? :)

Reply:
Groß und Kleinschreibung ist in diesem Punkt kein Problem. Die Methoden createRequest und updateBrands sind so definiert, dass sie alles "toLowerCase" Umwandeln. Damit brauchen wir uns damit nicht in verschiedenen Punkten in der Suche rumschlagen.

Reply:
Gut, darauf hattten wir uns ja jetzt geeinigt, dann kann ich das hier schließen.